### PR TITLE
fix(php): Factor common code

### PR DESCRIPTION
**Pull request by vargenau**
## Description

Some code is duplicated.

### Changes

Factor common code.
Remove useless local variables.

### CODE REVIEW

**Comment by vargenau** [comment]
Use master for these 4 files.

**Review by GMishx — CHANGES_REQUESTED** [review]

**Comment by vargenau** [comment]
Indentation fixed.

File: src/ununpack/ui/ui-reunpack.php [hunk 1]
```diff
@@ -167,21 +167,19 @@ function AgentAdd ($uploadpk, $Depends=NULL, $priority=0)
...
+        $result = pg_query($PG_CONN, $SQLInsert);
+        DBCheckResult($result, $SQLInsert, __FILE__, __LINE__);
+        $row = pg_fetch_assoc($result);
+        pg_free_result($result);
+        $SQLcheck = "SELECT job_pk FROM job WHERE job_upload_fk = '$uploadpk'"
+            . " AND job_name = '$Job_name' AND job_user_fk = '$user_fk';";
+        $result = pg_query($PG_CONN, $SQLcheck);
+        DBCheckResult($result, $SQLcheck, __FILE__, __LINE__);
+        $row = pg_fetch_assoc($result);
+        pg_free_result($result);
```
Comment (GMishx) [inline src/ununpack/ui/ui-reunpack.php:180]: Please fix the indentation
```suggestion
    if (empty($row)) {
      $result = pg_query($PG_CONN, $SQLInsert);
      DBCheckResult($result, $SQLInsert, __FILE__, __LINE__);
      $row = pg_fetch_assoc($result);
      pg_free_result($result);
      $SQLcheck = "SELECT job_pk FROM job WHERE job_upload_fk = '$uploadpk'"
        . " AND job_name = '$Job_name' AND job_user_fk = '$user_fk';";
      $result = pg_query($PG_CONN, $SQLcheck);
      DBCheckResult($result, $SQLcheck, __FILE__, __LINE__);
      $row = pg_fetch_assoc($result);
      pg_free_result($result);
```

**Review by GMishx — CHANGES_REQUESTED** [review]
Needs changes at 2 more locations.

**Comment by vargenau** [comment]
@GMishx 
Indentation fixed

**Review by GMishx — APPROVED** [review]

**Pull request merged**